spark-sql-application/src/test/scala/org/opensearch/sql/SQLJobTest.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql
+
+import org.scalatest.funsuite.AnyFunSuite
+import org.apache.spark.sql.{DataFrame, Row, SparkSession}
+import org.apache.spark.sql.types.{ArrayType, IntegerType, StringType, StructField, StructType}
+
+
+class SQLJobTest extends AnyFunSuite{
+
+  val spark = SparkSession.builder().appName("Test").master("local").getOrCreate()
+
+  // Define input dataframe
+  val inputSchema = StructType(Seq(
+    StructField("Letter", StringType, nullable = false),
+    StructField("Number", IntegerType, nullable = false)
+  ))
+  val inputRows = Seq(
+    Row("A", 1),
+    Row("B", 2),
+    Row("C", 3)
+  )
+  val inputDF: DataFrame = spark.createDataFrame(spark.sparkContext.parallelize(inputRows), inputSchema)
+
+  test("Test getJson method") {
+    // Define expected dataframe
+    val expectedSchema = StructType(Seq(
+      StructField("json", StringType, nullable = true)
+    ))
+    val expectedRows = Seq(
+      Row("{\"Letter\":\"A\",\"Number\":1}"),
+      Row("{\"Letter\":\"B\",\"Number\":2}"),
+      Row("{\"Letter\":\"C\",\"Number\":3}")
+    )
+    val expected: DataFrame = spark.createDataFrame(spark.sparkContext.parallelize(expectedRows), expectedSchema)
+
+    // Compare the result
+    val result = SQLJob.getJson(inputDF)
+    assertEqualDataframe(expected, result)
+  }
+
+  test("Test getSchema method") {
+    // Define expected dataframe
+    val expectedSchema = StructType(Seq(
+      StructField("column_name", StringType, nullable = false),
+      StructField("data_type", StringType, nullable = false)
+    ))
+    val expectedRows = Seq(
+      Row("Letter","string"),
+      Row("Number", "integer")
+    )
+    val expected: DataFrame = spark.createDataFrame(spark.sparkContext.parallelize(expectedRows), expectedSchema)
+
+    // Compare the result
+    val result = SQLJob.getSchema(spark, inputDF)
+    assertEqualDataframe(expected, result)
+  }
+
+  test("Test getList method") {
+    val name = "list"
+    // Define expected dataframe
+    val expectedSchema = StructType(Seq(
+      StructField(name, ArrayType(StringType, containsNull = true), nullable = true)
+    ))
+    val expectedRows = Seq(
```

Comment:
the expected in here is different with the description which include triple quote string, 

```
          "result" : [
            """{"name":"Tina","age":29,"city":"Bellevue"}""",
            """{"name":"Jane","age":25,"city":"London"}""",
            """{"name":"Mike","age":35,"city":"Paris"}"""
          ],
```
does triple quote string valid in json?

Reply:
After replacing `\"` with `'` in all json string, triple quotes is removed from index and json is valid. updated doc and description accordingly.